mythril/laser/ethereum/call.py

```diff
@@ -138,12 +141,18 @@ def get_callee_account(global_state, callee_address, dynamic_loader):
     return callee_account
 
 
-def get_call_data(global_state, memory_start, memory_size, pad=True):
+def get_call_data(
+    global_state: GlobalState,
+    memory_start: Union[int, BitVecNumRef, BoolRef],
```

Comment:
the memory_start and memory_size can both be symbolic right?

Reply:
Correct. I just noticed that with the new `Calldata` object from the develop branch the `pad` parameter is obsolete. Should I throw that out?

Reply:
Remove it :+1: